### PR TITLE
notmuch-addrlookup: 7 -> 9

### PR DESCRIPTION
**Pull request by flokli**
###### Motivation for this change
This version bump contains the following fixes:

v8:
 - Avoid a segmentation fault when notmuch_message_get_header() returns NULL. (Patch by Víctor M. Jáquez <vjaquez@igalia.com>).

v9:
 - Do not use the deprecated notmuch_query_count_messages_st function when using Notmuch 0.25. (Patch by Adam Ruzicka <a.ruzicka@outlook.com>.)
   We already had this patch in our tree.
 - Fix crash in when running queries and the program is build with Notmuch 0.25. (Patch by Joshua Krusell <joshua.krusell@v-dem.net>).
 - Fix conditional compilation when using Notmuch 0.25. (Patch by David Bremner <bremner@debian.org>).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by andir** [comment]
@GrahamcOfBorg build notmuch-addrlookup

File: pkgs/applications/version-management/git-and-tools/svn-all-fast-export/default.nix [hunk 1]
```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation {
     owner = "svn-all-fast-export";
     repo = "svn2git";
     rev = version;
-    sha256 = "0lhnw8f15j4wkpswhrjd7bp9xkhbk32zmszaxayzfhbdl0g7pzwj";
```
Comment (andir) [inline pkgs/applications/version-management/git-and-tools/svn-all-fast-export/default.nix]: Why is this in here?

**Review by andir — CHANGES_REQUESTED** [review]

File: pkgs/applications/version-management/git-and-tools/svn-all-fast-export/default.nix (hunk 1 above)
Comment (flokli) [reply pkgs/applications/version-management/git-and-tools/svn-all-fast-export/default.nix]: Whoops, that was a mistake. fixed!

**Comment by andir** [comment]
@GrahamcOfBorg build notmuch-addrlookup

**Comment by flokli** [comment]
@andir The jemalloc issue on aarch64 looks pretty unrelated.

I don't see a reason why it shouldn't work on darwin. Could you give @GrahamcOfBorg another shot?

**Comment by andir** [comment]
@GrahamcOfBorg build notmuch-addrlookup

**Review by andir — APPROVED** [review]

**Pull request merged**